manager/frontend/src/containers/Feed.js

```diff
@@ -5,10 +5,12 @@ import _ from 'lodash';
 import Navbar from '../components/Navbar/Navbar';
 import PostCreator from '../components/PostCreator/PostCreator';
 import PostSorter from '../components/PostSorter/PostSorter';
-import Posts from '../components/Posts/Posts';
+import Post from '../components/Posts/Post/Post';
```

Comment:
why do you need to import this?

Reply:
So that an array of Post can be map'd in the Feed component

Reply:
sure, we can refactor later if we need to